### PR TITLE
converter detected page layout develop

### PR DESCRIPTION
**Pull request by yoouung**
- add converter detected result page
- TODO: rename 오류유형s in table

### CODE REVIEW

**Comment by yoouung** [comment]
### page layout
image

**Comment by yoouung** [comment]
### processing Converter until detection (this page)

Uploading 5f2258a4-f4e2-45bd-a3c1-6b0fe8d8717b.mp4…

File: dt-demo-be/app_lab.py [hunk 1]
```diff
@@ -132,6 +132,46 @@ def converter_file_list():
...
+
+    if result == "SUCCESS":
+        # TODO: handle multiple files
+        doc_list.pop(0) # Assuming that there's only one file
+
+    return jsonify({'result': f"{result}"})
+
+# TODO: update detReportInfo
+@app.route("/api/converter/detreport", methods=["GET"])
+def converter_get_det_report():
```
Comment (doorooful) [inline dt-demo-be/app_lab.py:165]: Can you modify this function as follows?
Please use `/api/converter/detreport` endpoint for converter-detection page.

``` sample
    app.logger.info("API request to get detection report of converter")
    doc_list[0].run()
    db_conn = dbconn.connect_to_db()
    doc_list[0].fetchDetReportInfo(db_conn)
    doc_list[0].fetchConvReportInfo(db_conn, "word") # Conversion option is automatically set as 'word'
    db_conn.close()
    
    # TODO: Refactor this function 
    return doc_list[0].getConvReportInfo()
```

File: dt-demo-fe/src/pages/converter/detected.js [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    );
+    const [highlightedContents, sethighlightedContents] = useState("");
+    const [details, setDetails] = useState([]);
+
+    const getResult = async () => {
+        try {
+            const detReport_data = JSON.parse(localStorage.getItem("converter_detData"));
+            const { sentences } = detReport_data;
+
+            const convReport_data = await useApi.get("/converter/convreport");
```
Comment (doorooful) [inline dt-demo-fe/src/pages/converter/detected.js:101]: Please use `detData` for `ConverterDetectionResult` component.
Also, please modify api endpoint url.

File: dt-demo-fe/src/pages/converter/detected.js [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+                );
+                if (!matches) continue;
+
+                const error = matches ? matches[2] : null;
+                const expected = converted_highlighted_sentences.match(
+                    /<mark id='([^']+)'>([^<]+)<\/mark>/
+                )[2];
+
+                // TODO: modify this part after the duplication detection module is implemented
+                // let conv_final = converted_highlighted_sentences;
```
Comment (doorooful) [inline dt-demo-fe/src/pages/converter/detected.js:145]: Please remove unnecessary part.
Or use multi-line comment `/* */` with mentioning that this part would be used as sample code:
``` sample.js
// TODO: add detection module handling part
/* sample code
let conv_final = converted_highlighted_sentences;
 if (converted_highlighted_sentences.includes("<del>")) {
     cstyle += ";text-decoration: line-through;";
     conv_final = converted_highlighted_sentences.replace(/<del>/g, "");
}
*/
```

File: dt-demo-fe/src/Router.js [hunk 4]
```diff
@@ -20,13 +21,14 @@ const Router = () => {
...
                     <Route index element={<FilterMain />} />
-                    <Route path="result" element={<FilterResult />} />
+                    <Route path="filtered" element={<FilterResult />} />
                 </Route>
 
                 {/* Converter */}
                 <Route path="converter" element={<Converter />}>
                     <Route index element={<ConverterMain />} />
-                    <Route path="conversion" element={<ConverterConversion />} />
+                    {/* TODO: rename path name */}
```
Comment (doorooful) [inline dt-demo-fe/src/Router.js:30]: Is this line necessary?

**Review by doorooful — CHANGES_REQUESTED** [review]
Please check my comments! If you have any suggestions please let me know :)

**Comment by yoouung** [comment]
### update for detecting multiple errors in one sentence
image

File: dt-demo-fe/src/pages/converter/detected.js [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+    };
+
+    const HandleDelete = () => {
+        alert("삭제");
+    };
+
+    const navigate = useNavigate();
+    const handleConvert = async () => {
+        try {
+            // TODO: update below after convreport api is enhanced
```
Comment (doorooful) [inline dt-demo-fe/src/pages/converter/detected.js:187]: Can you use multi line comment on sample code? `/* */`

**Review by doorooful — APPROVED** [review]
Please update upon comments ;)

File: dt-demo-fe/src/pages/converter/detected.js (hunk 5 above)
Comment (yoouung) [reply dt-demo-fe/src/pages/converter/detected.js:187]: updated :) thanks!

**Pull request merged**